test/lib/logger.test.js

```diff
@@ -0,0 +1,169 @@
+const chai = require('chai')
+const spies = require('chai-spies')
+const proxyquire = require('proxyquire')
+const path = require('path')
+
+const expect = chai.expect
+chai.use(spies)
+
+const CORRELATION_ID = 'CORRELATION_ID'
+const MODULE_NAME = path.basename(path.resolve())
+const MOCK_DATE = 1558625994627
+
+const LOG_LEVELS = require('../../lib/log-levels')
+
+const { Logger } = proxyquire('../../lib/logger', {
+  '@first-lego-league/ms-correlation': {
+    getCorrelationId () {
+      return CORRELATION_ID
+    }
+  }
+})
+
+describe('Logger', () => {
+  it('returns a logger even if not called as a constructor', () => {
+    expect(Logger() instanceof Logger).to.equal(true)
+  })
+
+  it('has a default level of DEBUG', () => {
+    expect(Logger().logLevel).to.equal(LOG_LEVELS.DEBUG)
+  })
+
+  describe('set log level', () => {
```

Comment:
redundant describe. it's contains to tests regrading a relatively simple process/operation.